src/tutorials/0004-mutable-file-system/10.md

```diff
@@ -8,23 +8,20 @@ MFS has a [`files.read`](https://github.com/ipfs/interface-js-ipfs-core/blob/mas
 The method takes this format:
 
 ```js
-await ipfs.files.read(path, [options])
+ipfs.files.read(path, [options])
 ```
 
 The `path` provided is the path of the file to read, and it must point to a file rather than a directory.
 
-The `files.read` method returns a Buffer, which can be converted to a string using the method `toString('utf8')`. For example:
+The `files.read` method returns an Async Iterable, that iterates over the file's chunks of data, i.e. Buffers, which can be converted to a string using the method `toString('utf8')`. For example:
 
 ```js
-let bufferedContents = await ipfs.files.read('/directory/some-file.txt')  // a buffer
-let contents = bufferedContents.toString('utf8') // a string
-```
-
-or
+// the concat variable is globally available (just like ipfs)
```

Comment:
We need to introduce `concat` & `it-concat` before using it.

Reply:
updated here: https://github.com/ProtoSchool/protoschool.github.io/pull/381/commits/771fa666432f40e0f4363d0d695bd6246db5e610